docs/ccoctl.md

```diff
@@ -335,4 +335,89 @@ For alibaba cloud,  the CCO utility (`ccoctl`) binary will create credentials Se
    ```
    where:
    - `name` is the name used to tag any cloud resources that are created for tracking. 
-   - `region` is the Alibaba Cloud region in which cloud resources will be created.   
\ No newline at end of file
+   - `region` is the Alibaba Cloud region in which cloud resources will be created.   
+
+## Nutanix
+
+This is a guide for using manual mode on Nutanix, for more info about manual mode, please refer to [cco-mode-manual](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).
+
+For Nutanix,  the CCO utility (`ccoctl`) binary will create credentials Secret manifests for the OpenShift installer.
+
+### Procedure
```

Comment:
Before doing all this, I assume the same prereq steps of getting the release image and setting up the `ccoctl`. Do they also need to do anything on the cloud provider end (such as in Alibaba procedure)?

Reply:
Need to look further into Alibaba and how they're going about it. We are mostly taking credentials requests and creating secret manifests with the same secret for all the corresponding credentials requests.

Reply:
No rush from me, but adding this comment for tracking (and to remind myself where we are :slightly_smiling_face:):

Can add `docs-approved` label once this question is resolved

Reply:
It is in the sense that the workflow is manual entirely via ccoctl to extract credential requests via oc and generate corresponding secret manifests. We have added a prerequisite step to ensure user prepares a ccoctl binary  https://github.com/openshift/cloud-credential-operator/pull/450/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR349

Reply:
@thunderboltsid i think the question is are there any pre-reqs that a customer has to complete prior to running `ccoutl`.  each cloud has it's own cloud specific pre-reqs that have to be completed prior to generating the manifests.

Reply:
@rvanderp3 yes, exactly what I am looking for.

@thunderboltsid is there anything they need to do analogous to items [2&3 here](https://github.com/openshift/cloud-credential-operator/blob/master/docs/ccoctl.md#prerequisite)? Or anything else the user has to do like creating specific accounts etc. (especially on the Nutanix side) to get this ready aside from the usual `ccoctl` binary set up?

Reply:
@jeana-redhat @rvanderp3 I see. No, there are no extra steps that need to be performed other than the ones already mentioned in pre-requisites.